backends/transforms/decompose_sdpa.py

```diff
@@ -1,8 +1,11 @@
-# Copyright (c) Qualcomm Innovation Center, Inc.
-# All rights reserved
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
For Qualcomm folks (@shewu-quic , @chunit-quic, @haowhsu-quic) since the decompose sdpa is helpful for other backends, mind us moving it to a central place so we can reuse?

Reply:
Sure, that's fine. Good to see this!